### ISSUE TITLE
Can't open file minilight.ini

### ISSUE THREAD

**Issue by nunohora** [golemfactory/homebrew-golem#5]
I'm running this on Sierra v10.12.3

When I try to run `golemapp` I get the following error:

```bash
:dev nunohora$ golemapp

Failed to import scrypt. This is not a fatal error but does
mean that you cannot create or decrypt privkey jsons that use
scrypt

WARNING  [golem.appconfig                    ] Can't open file /Users/nunohora/Library/Application Support/golem/SimpleEnv/minilight.ini: [Errno 2] No such file or directory: '/Users/nunohora/Library/Application Support/golem/SimpleEnv/minilight.ini'
Traceback (most recent call last):
  File "/Users/hawk/work/workspaces/golem-project/golem/golemapp.py", line 98, in <module>
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
  File "/Users/hawk/work/workspaces/golem-project/golem/golemapp.py", line 68, in start
  File "/Users/hawk/work/workspaces/golem-project/golem/gui/startapp.py", line 125, in start_app
  File "/Users/hawk/work/workspaces/golem-project/golem/gui/startapp.py", line 70, in start_client
  File "/Users/hawk/work/workspaces/golem-project/golem/golem/client.py", line 126, in __init__
  File "/Users/hawk/work/workspaces/golem-project/golem/golem/transactions/ethereum/ethereumtransactionsystem.py", line 30, in __init__
  File "/Users/hawk/work/workspaces/golem-project/golem/golem/ethereum/client.py", line 19, in __init__
  File "/Users/hawk/work/workspaces/golem-project/golem/golem/ethereum/node.py", line 73, in __init__
AttributeError: 'NoneType' object has no attribute 'group'
Failed to execute script golemapp
```

**Comment by k06a** [comment]
Having exactly the same problem

**Comment by k06a** [comment]
Reinstall helped to solve this issue:
```
brew untap ethereum/ethereum
brew tap golemfactory/golem
brew reinstall golem
```

**Comment by k06a** [comment]
But second launch gives:
```
$ golemapp

Failed to import scrypt. This is not a fatal error but does
mean that you cannot create or decrypt privkey jsons that use
scrypt

Traceback (most recent call last):
  File "/Users/hawk/work/workspaces/golem-project/golem/golemapp.py", line 98, in <module>
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 697, in main
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
  File "/Users/hawk/work/workspaces/golem-project/golem/golemapp.py", line 68, in start
  File "/Users/hawk/work/workspaces/golem-project/golem/gui/startapp.py", line 125, in start_app
  File "/Users/hawk/work/workspaces/golem-project/golem/gui/startapp.py", line 70, in start_client
  File "/Users/hawk/work/workspaces/golem-project/golem/golem/client.py", line 75, in __init__
  File "/Users/hawk/work/workspaces/golem-project/golem/golem/client.py", line 842, in __lock_datadir
IOError: Data dir /Users/k06a/Library/Application Support/golem/default used by other Golem instance
Failed to execute script golemapp
```

**Comment by lal00pe** [comment]
Did you find a solution for this issue ?

**Comment by mfranciszkiewicz** [comment]
It seems that Golem didn't quit gracefully. In this case, you can simply remove the lock file located at
`~/Library/Application\ Support/golem/default/LOCK`

**Comment by PanamaMike** [comment]
I don't have the minilight.ini.  I tried deleting the lock, but I'm still getting the same error.
2017-07-24 19:46:55 WARNING  golem.appconfig                     Can't open file C:\Users\LocalUser\AppData\Local\golem\golem\SimpleEnv\minilight.ini: [Errno 2] 
No such file or directory:u'C:\\Users\\LocalUser\\AppData\\Local\\golem\\golem\\SimpleEnv\\minilight.ini'

**Comment by maaktweluit** [comment]
The minilight issue has been solved in golemfactory/golem#1296

The other error was related to a LOCK file as mentioned by @mfranciszkiewicz 

Please re-open or create a new issue if you still have these errors.
